config/kubernetes/sig-docs/teams.yaml

```diff
@@ -210,21 +206,22 @@ teams:
     members:
     - bradamant3
     - bradtopol
-    - chenopis
     - cody-clark
-    - jaredbhatti
+    - gochist
     - jimangel
     - kbarnard10
+    - kbhawkey
+    - makoscafee
     - mistyhacks
-    - Rajakavitha1
+    - rajakavitha1
```

Comment:
Would prefer to keep this as `Rajakavitha1` since that's how the GitHub username is structured: https://github.com/Rajakavitha1